### PR TITLE
tests: container creation with --global-name

### LINKED ISSUE #138: Containers with nice names
Test containers with `__NEOFS_NAME` and `__NEOFS__ZONE` attributes. Check availability of container name in NNS contract (TXT record should return string with base58 encoded container ID).

Testcases:
1. Container created with `__NEOFS_NAME=foo` attribute available in NNS with `foo.container` domain.
2. Container with the same name can't be created twice.
3. Container can be created with name and `__NEOFS__ZONE=container`. 
4. Container can't be created with any other zone.
5. Container can' be created with zone but without name.

### PR DESCRIPTION
**Pull request by evgeniiz321**
closes #138

### CODE REVIEW

**Comment by evgeniiz321** [comment]
@roman-khimov in the original ticket https://github.com/nspcc-dev/neofs-testcases/issues/138, this attribute - `__NEOFS__ZONE` is also mentioned, do we want to have this attribute in the container create cli command?

**Comment by roman-khimov** [comment]
I'd not bother, many things would break if we're to touch it.

**Review by roman-khimov — APPROVED** [review]

File: pytest_tests/tests/container/test_container.py [hunk 1]
```diff
@@ -211,3 +211,35 @@ def test_container_deletion_while_sn_down(
 
         with allure.step("Previously stopped node should return GC MARKED for the created object"):
             object_should_be_gc_marked(self.neofs_env, node_to_stop, cid, oid)
+
+    def test_container_global_name(self, default_wallet, simple_object_size):
```
Comment (roman-khimov) [inline pytest_tests/tests/container/test_container.py:215]: This test should be version-dependent, it fails with 0.43.0.

Comment (evgeniiz321) [reply pytest_tests/tests/container/test_container.py:215]: fixed

**Pull request merged**